plugins/dev.ionfusion.xtext/src/dev/ionfusion/ion/plugin/validation/IonJavaValidator.java

```diff
@@ -1,7 +1,7 @@
 // Copyright Ion Fusion contributors. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package dev.ionfusion.ion.plugin.validation;
+package software.amazon.ionxtext.plugin.validation;
```

Comment:
```suggestion
package dev.ionfusion.ion.plugin.validation;
```

This is still our code, needs to match directory structure.

Reply:
Good catch! Missed this interaction with my directory renames.